secret/secret_test.go

```diff
@@ -277,19 +268,6 @@ func TestGenerateSecretsWithNoSecrets(t *testing.T) {
 		generateSSLCerts = origGenerateSSLCerts
 	}()
 
-	t.Run("Manifest with no secrets doesn't change", func(t *testing.T) {
```

Comment:
We no longer need to assert this?

Reply:
That test is passing in `nil` as the `secrets` and `updates`, which aren't really valid values to begin with.

We could add a real test that an empty manifest doesn't change anything, but feels like low-value to me.